UPGRADE.md

```diff
@@ -2,6 +2,15 @@
 
 ## dev-develop
 
+### Renamed ContentBundle to WebspaceBundle
+
+Following things have changed:
+
+* The bundle name changed from `SuluContentBundle` to `SuluWebspaceBundle`
+* All services or parameters prefixed with `sulu_content` will now use the prefix `sulu_webspace`
```

Comment:
Should that also be the case for all the twig functions?

Reply:
When thinking about it... That would mean that it results in `sulu_webspace_load` to load a page :see_no_evil: Maybe `sulu_page` would be really the better prefix?

Reply:
will be done in an own PR